printer_mods/sloscotty/SW_Wall_Plate/README.md

```diff
@@ -0,0 +1,38 @@
+# Switchwire Wall Plate
+This mod is for pure aesthetics.  A Voron Switchwire logo plate to install on the back wall of your printer's enclosure.
+
+(Or optionally on the front grill if no enclosure - but only if you're not already using the LCD mount...)
+
+![](Images/SW_backwall_plate.jpg)
+
+Note: Overall size is about 225x50 (about 10mm thick)
+
+## BOM
+* (4) M3x8mm BHCS
+* (4) M3xD5.0xH4.0 heat-set inserts
+
+![](Images/printed_pieces.jpg)
+
+## Directions
+1.  Install inserts in the WITCHWIRE_base
+
+![](Images/base_inserts.jpg)
+
+2. Use backplate_and_drillguide to drill holes in your back panel
+
+3. Glue badges into respective recesses on the base (no need to glue base to backplate)
+    
+    Note: badge tolerances are tight - you might need to sand a bit off the edges of the badges.
+
+![](Images/assembled1.jpg)
+4. Attach to back panel with (4) M3x8 screws
+
+(Optional: to use on front grill, glue all parts together and glue the optional nolcd_grill_mount to the center top of the backplate.)
```

Comment:
Again, no glue, please find a different way to hold the parts together.

Reply:
Is it OK to ask them to acetone weld them together?  That is actually what I do.  The pieces are separate, and there is not space for screws for the two small badges.  I would have to redesign the parts to ask for a filament change.

Reply:
Filament changes are not something even I am comfortable with, FWIW.

Reply:
@FHeilmann is acetone welding parts together acceptable?  I don't know if we have any official weld mounts, but I thought I saw them somewhere.

Reply:
I wouldn't ask someone to acetone weld a structural part, but this is a purely cosmetic mod.

Reply:
I updated the README.md to allow the option of using vhb tape or acetone to affix the parts.  I hope this is sufficient. Thx.

Reply:
@KiloQubit VHB is fine, my other suggestion would have been to do a press-fit so the detail pieces can slot into place.

Reply:
I will try the press-fit.  Good idea!  It will require a slight redesign.  Quick question for @KiloQubit  :  If I replace files in my repository (the .scad and .STL), can they have the same names - or should they be different?